thing-management.md

```diff
@@ -115,36 +101,46 @@ Described in Node Provisioning Information Type Registry 3.1.2.4.
 
 Can be one of the following:
 
-- 1: Z-Wave is supported
-- 2: Z-Wave Long Range is supported
+- 0: Z-Wave is supported
+- 1: Z-Wave Long Range is supported
 
 #### `name`
 
 The name of the device being upserted in the NPL.
+The Name:
+- must be UTF-8 encoded
+- must not contain any appended termination characters
+- may contain the dot
+- must not contain the underscore character “_”.
+- Each name sub-string (separated by the dot character “.”) must not end with the dash character “-”.
+
+Device name is case sensitive.
+The combined Name and Location strings must not be longer than 62 characters.
+
 
 #### `location`
 
 Location assigned to a device.
+The Location:
+- must be UTF-8 encoded
+- must not contain any appended termination characters
+- may contain the dot
+- must not contain the underscore character “_”.
+- Each location sub-string (separated by the dot character “.”) must not end with the dash character “-”.
+
```

Comment:
can you provide som reges form so someone could take a look how it should be formatted exactly ? liek XX.\d{2} etc